Gemfile

```diff
@@ -41,44 +40,45 @@ gem "acts_as_tree",                   "~>2.1.0"  # acts_as_tree needs to be requ
 # See miq_expression_spec Date/Time Support examples.
 # https://github.com/jeremyevans/ruby-american_date
 gem "american_date"
```

Comment:
@gtanzillo @dclarizio @chessbyte I think it's still needed although it's worth determining where raw date strings would be different if we dropped american_date.

So, american_date is for ruby land parsing of datetime related strings.  In ruby 1.9, they changed date parsing to  swap the day and month from american style to "everywhere else".  Note, even if we used iso8601, we'd still have to determine what's the day/month for incoming strings before we store it in iso8601.

## Ruby 1.9+
```ruby
irb(main):011:0> Date.parse('01/02/2003').month
=> 2
irb(main):012:0> Date.parse('01/02/2003').day
=> 1
irb(main):013:0> Date.parse('01/02/2003').iso8601
=> "2003-02-01"
```

## Ruby 1.9+ with american date
```ruby
irb(main):001:0> Date.parse('01/02/2003').month
=> 1
irb(main):002:0> Date.parse('01/02/2003').day
=> 2
irb(main):003:0> Date.parse('01/02/2003').iso8601
=> "2003-01-02"

```
See: https://github.com/jeremyevans/ruby-american_date/blob/master/spec/american_date_spec.rb